README.md

```diff
@@ -40,7 +40,8 @@ The [OCI Image Media Types](media-types.md) document is a starting point to unde
 The high-level components of the spec include:
 
 * An archival format for container images, consisting of an [image manifest](manifest.md), an [image layout](image-layout.md), a set of [filesystem layers](layer.md), and [image configuration](config.md) (base OCI layer)
-* A process of hashing the image format for integrity and content-addressing (base OCI layer)
+* A [process of hashing the image format](descriptor.md) for integrity and content-addressing (base OCI layer)
```

Comment:
How about "a process of referencing container images by a cryptographic hash of their content", which accords better with the relevant table layers ("Use of Hash as Content Addressable name for immutable containers")